test/common.go

```diff
@@ -23,7 +23,7 @@ import (
 func deployTerraform(t *testing.T, workingDir string, vars map[string]interface{}) {
 	var terraformOptions *terraform.Options
 
-	if test_structure.IsTestDataPresent(t, test_structure.FormatTestDataPath(workingDir, "TerraformOptions.json")) {
+	if test_structure.IsTestDactaPresent(t, test_structure.FormatTestDataPath(workingDir, "TerraformOptions.json")) {
```

Comment:
Are you deliberately introducing a typo? :)

Reply:
heh no